### PR TITLE
Refactoring and removing the support for the sched_switch

### PR DESCRIPTION
**Pull request by MariaSandru27**
tracepoint in favour of a generalization for all tracepoints

### CODE REVIEW

File: OrbitGl/CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@
 cmake_minimum_required(VERSION 3.15)
 
 project(OrbitGl CXX)
-add_library(OrbitGl STATIC)
+add_library(OrbitGl STATIC TracepointCustom.h)
```
Comment (dimitry-) [inline OrbitGl/CMakeLists.txt:8]: move this to target_sources below

**Review by dimitry- — APPROVED** [review]

File: OrbitGl/TracepointCustom.h [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_TRACEPOINTCUSTOM_H
+#define ORBIT_TRACEPOINTCUSTOM_H
+
+#include "tracepoint.pb.h"
+
+namespace internal {
```
Comment (danielfenner) [inline OrbitGl/TracepointCustom.h:10]: By convention stuff from internal namespaces should only occur in in the header where it is defined (or the related .cc). If you need to make these functors more widely  available remove the "namespace internal {".

Comment (MariaSandru27) [reply OrbitGl/TracepointCustom.h:10]: Sure. Thank you so much! I will modify it right now.

**Review by danielfenner — APPROVED** [review]
LGTM!

**Pull request merged**